.taskcluster.yml

```diff
@@ -53,6 +56,31 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("test_mozilla_central")}
```

Comment:
We should use a cache for the repository clone, so we don't reclone from scratch every time.
Here's an example: https://github.com/mozilla/bugbug/blob/ff9f8a78d1f47d2067c130fe5a791a360fac7413/infra/data-pipeline.yml#L13-L47.

Reply:
I tried to add scopes, but I get the following error: https://github.com/mozilla/rust-code-analysis/commit/aaf9eede020b6212493e3639be156af6da35033f#commitcomment-43107625

Reply:
Need to wait on https://github.com/mozilla/community-tc-config/pull/352.